2019/12/26 11:18:53 [crit] 2934#2934: *1 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 103.214.131.114, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "ec2-18-212-16-168.compute-1.amazonaws.com"
2019/12/26 11:18:53 [error] 2934#2934: *1 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "ec2-18-212-16-168.compute-1.amazonaws.com"
2019/12/26 11:20:25 [crit] 2934#2934: *3 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 103.214.131.114, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "ec2-18-212-16-168.compute-1.amazonaws.com"
2019/12/26 11:20:25 [error] 2934#2934: *3 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "ec2-18-212-16-168.compute-1.amazonaws.com"
2019/12/26 11:21:34 [error] 2934#2934: *5 open() "/home/ubuntu/hello_django/static/admin/css/fonts.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/fonts.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/"
2019/12/26 11:22:03 [error] 2934#2934: *5 open() "/home/ubuntu/hello_django/static/admin/css/base.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/base.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/admin/login/?next=/admin/"
2019/12/26 11:22:03 [error] 2934#2934: *10 open() "/home/ubuntu/hello_django/static/admin/css/responsive.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/responsive.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/admin/login/?next=/admin/"
2019/12/26 11:22:03 [error] 2934#2934: *11 open() "/home/ubuntu/hello_django/static/admin/css/login.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/login.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/admin/login/?next=/admin/"
2019/12/26 11:22:37 [error] 2934#2934: *10 open() "/home/ubuntu/hello_django/static/admin/css/login.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/login.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/admin/login/?next=/admin/"
2019/12/26 11:22:37 [error] 2934#2934: *11 open() "/home/ubuntu/hello_django/static/admin/css/base.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/base.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/admin/login/?next=/admin/"
2019/12/26 11:22:37 [error] 2934#2934: *5 open() "/home/ubuntu/hello_django/static/admin/css/responsive.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/responsive.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/admin/login/?next=/admin/"
2019/12/26 11:22:49 [error] 2934#2934: *5 open() "/home/ubuntu/hello_django/static/admin/css/fonts.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/fonts.css HTTP/1.1", host: "ec2-18-212-16-168.compute-1.amazonaws.com", referrer: "http://ec2-18-212-16-168.compute-1.amazonaws.com/"
2019/12/26 12:08:34 [error] 2934#2934: *14 open() "/home/ubuntu/hello_django/static/admin/css/fonts.css" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET /static/admin/css/fonts.css HTTP/1.1", host: "18.212.16.168", referrer: "http://18.212.16.168/"
2019/12/26 12:09:05 [crit] 2934#2934: *14 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 103.214.131.114, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "18.212.16.168"
2019/12/26 12:09:05 [error] 2934#2934: *14 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 103.214.131.114, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "18.212.16.168"
2019/12/26 12:59:27 [crit] 2934#2934: *18 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 143.137.150.128, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "18.212.16.168:80"
2019/12/26 12:59:27 [error] 2934#2934: *18 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 143.137.150.128, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "18.212.16.168:80"
2019/12/26 13:06:11 [crit] 2934#2934: *20 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET http://example.com/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "example.com"
2019/12/26 13:06:11 [error] 2934#2934: *20 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET http://example.com/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "example.com"
2019/12/26 13:06:11 [crit] 2934#2934: *22 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET http://169.254.169.254/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "169.254.169.254"
2019/12/26 13:06:11 [error] 2934#2934: *22 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET http://169.254.169.254/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "169.254.169.254"
2019/12/26 13:06:11 [crit] 2934#2934: *24 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET http://[::ffff:a9fe:a9fe]/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:11 [error] 2934#2934: *24 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET http://[::ffff:a9fe:a9fe]/ HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:11 [crit] 2934#2934: *26 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET http://169.254.169.254/latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/latest/dynamic/instance-identity/document", host: "169.254.169.254"
2019/12/26 13:06:11 [error] 2934#2934: *26 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET http://169.254.169.254/latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/latest/dynamic/instance-identity/document", host: "169.254.169.254"
2019/12/26 13:06:12 [crit] 2934#2934: *28 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET http://[::ffff:a9fe:a9fe]/latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/latest/dynamic/instance-identity/document", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:12 [error] 2934#2934: *28 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET http://[::ffff:a9fe:a9fe]/latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/latest/dynamic/instance-identity/document", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:12 [crit] 2934#2934: *30 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "example.com"
2019/12/26 13:06:12 [error] 2934#2934: *30 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "example.com"
2019/12/26 13:06:12 [crit] 2934#2934: *32 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "169.254.169.254"
2019/12/26 13:06:12 [error] 2934#2934: *32 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "169.254.169.254"
2019/12/26 13:06:12 [crit] 2934#2934: *34 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:12 [error] 2934#2934: *34 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:12 [crit] 2934#2934: *36 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET /latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/latest/dynamic/instance-identity/document", host: "169.254.169.254"
2019/12/26 13:06:12 [error] 2934#2934: *36 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET /latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/latest/dynamic/instance-identity/document", host: "169.254.169.254"
2019/12/26 13:06:12 [crit] 2934#2934: *38 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 44.225.84.206, server: example.com, request: "GET /latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/latest/dynamic/instance-identity/document", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:06:12 [error] 2934#2934: *38 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 44.225.84.206, server: example.com, request: "GET /latest/dynamic/instance-identity/document HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/latest/dynamic/instance-identity/document", host: "[::ffff:a9fe:a9fe]"
2019/12/26 13:30:28 [crit] 2934#2934: *50 connect() to unix:/home/ubuntu/hello_django/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 177.52.26.8, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock:/", host: "18.212.16.168:80"
2019/12/26 13:30:28 [error] 2934#2934: *50 open() "/home/ubuntu/hello_django/static/500.html" failed (2: No such file or directory), client: 177.52.26.8, server: example.com, request: "GET / HTTP/1.1", upstream: "http://unix:/home/ubuntu/hello_django/run/gunicorn.sock/", host: "18.212.16.168:80"
